actor/v2action/service_broker_test.go

```diff
@@ -0,0 +1,130 @@
+package v2action_test
+
+import (
+	"errors"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	. "code.cloudfoundry.org/cli/actor/v2action"
+	"code.cloudfoundry.org/cli/actor/v2action/v2actionfakes"
+	"code.cloudfoundry.org/cli/api/cloudcontroller/ccv2"
+)
+
+var _ = Describe("Service Broker", func() {
+	var (
+		actor                     *Actor
+		fakeCloudControllerClient *v2actionfakes.FakeCloudControllerClient
+	)
+
+	BeforeEach(func() {
+		fakeCloudControllerClient = new(v2actionfakes.FakeCloudControllerClient)
+		actor = NewActor(fakeCloudControllerClient, nil, nil)
+	})
+
+	Describe("CreateServiceBroker", func() {
+		var (
+			spaceGUID     string
+			executeErr    error
+			warnings      Warnings
+			serviceBroker ServiceBroker
+		)
+
+		JustBeforeEach(func() {
+			serviceBroker, warnings, executeErr = actor.CreateServiceBroker("broker-name", "username", "password", "https://broker.com", spaceGUID)
+		})
+
+		When("there are no errors", func() {
+			BeforeEach(func() {
+				spaceGUID = ""
+				fakeCloudControllerClient.CreateServiceBrokerReturns(ccv2.ServiceBroker{GUID: "some-broker-guid"}, nil, nil)
+			})
+
+			It("creates the service broker with the correct values", func() {
+				Expect(fakeCloudControllerClient.CreateServiceBrokerCallCount()).To(Equal(1))
+				brokerName, username, password, url, space := fakeCloudControllerClient.CreateServiceBrokerArgsForCall(0)
+				Expect(brokerName).To(Equal("broker-name"))
+				Expect(username).To(Equal("username"))
+				Expect(password).To(Equal("password"))
+				Expect(url).To(Equal("https://broker.com"))
+				Expect(space).To(Equal(""))
+
+				Expect(warnings).To(BeNil())
+				Expect(executeErr).NotTo(HaveOccurred())
+				Expect(serviceBroker.GUID).To(Equal("some-broker-guid"))
+			})
+		})
+
+		When("there are warnings", func() {
```

Comment:
We have a comment further down in the PR review about this, but we generally don't have dedicated contexts just for Warnings.

Reply:
We will remove the context dedicated to warnings.